src/SymbolCollector.Core/SymbolCollector.Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net9.0</TargetFrameworks>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
Is this needed for the bump?

Reply:
no ... not at all ... reverting that change ... keeping it to the package update only